resources/public/api/conf/2.0/errors.raml

```diff
@@ -211,8 +211,8 @@ traits:
           application/json:
             type: types.errorResponse
             examples:
-              rule_TaxYearNotSupported:
-                description: The specified taxYear is not supported. The taxYear specified is before the minimum tax year value.
+              ruleTaxYearNotSupported:
+                description: The specified tax year is not supported. Either the tax year specified is before the minimum tax year value, or it is after the maximum tax year value.
```

Comment:
It's quite a long message, could it be made multi-line with the | character? i.e.
``
                description: |
                  The specified tax year is not supported. Either the tax year specified is before the minimum tax year value,
                  or it is after the maximum tax year value.
```